src/main/java/org/folio/rest/core/BaseApi.java

```diff
@@ -4,16 +4,25 @@
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
 import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static org.folio.rest.tools.utils.TenantTool.tenantId;
```

Comment:
Done. I believe it could be beneficial for both maintainers and contributors if you could point to any specific documents, guides, or configuration files that outline the preferred formatting and conventions you are using. Currently, `CONTRIBUTING.md` only links to the generic FOLIO guidelines.